### DISCUSSION TITLE
Support for tags in sonarr?

### DISCUSSION THREAD

**Discussion by arpitgupta** [activexray/Doplarr#40] [Q&A]
Any plans on adding support for selecting tags in sonarr? One of the features i use in Requestrr.

Edit: Same for language.

**Comment by activexray** [comment]
Can you explain this a bit more? As in, you want a dialog box to add in tags or just select from existing tags? I haven't seen how it looks on requestrr.

**Comment by arpitgupta** [comment]
So in requesterr for sonarr when you define categories you can select what tags you want associated with that category. Since in Doplarr all that is selected as we interact with the bot then I would like the ability to add specific tags.


Same for language.

I also have different root folders for different profiles I want to work with. So ideally I would want the ability to be able to select all these every time i want to add content.

Having some sort of a shortcurt like categories in requrestrr would be best. Does not have to be via UI, if they can be defined in some sort of config/json etc that works as well. Not sure if want to select all these options every time i add something :).

**Reply by activexray** [reply]
There's a selection for language now in v3 (assuming you don't have a default set in the config). I have to think about categories a bit more, I think. Having them be a combo of configuration settings does make sense. Overseerr provides that kind of functionality too, but right now I only have exposed selecting 4k or not 4k. It might make sense to just select which server to request with since that does what you mention of setting rootfolder/language/quality - otherwise I have to figure out how to recreate that in the "vanilla" backends.

The multiple rootfolder thing I haven't even considered, and that probably breaks something since I'm just doing
```clj
(def rootfolder (delay (a/<!! (utils/request-and-process-body GET #(get (first %) "path") "/rootfolder"))))
```
More to test, I suppose.

**Comment by arpitgupta** [comment]
ya i looked at overseer and it honestly is overkill for what i want. Also it also does not have concept of categories. I was told for all the different categories i want I should add the same sonarr/radarr again. 

For example i have the following categories set both for radarr and sonarr in Requestrr. Hindi language selects a different root folder as I want a different library for that, all others go to the same root folder.
1080p
720+
4k
Hindi

**Comment by activexray** [comment]
I opened an issue to track this, I like the idea though. It shouldn't be too hard to implement.

**Reply by Eyeball753** [reply]
Has the tags thing been implemented? I also used this in Requestrr to send a tag along to Radarr/Sonarr to only download x265 content. I looked around in the issues and didn't see anything related to it, but I could've missed it.

**Reply by activexray** [reply]
I'm tracking in #48. Haven't had much time to get to it.
